core/domain/summary_services.py

```diff
@@ -63,6 +63,16 @@ def get_displayable_exp_summary_dicts_matching_ids(exploration_ids):
     displayable_exp_summaries = []
     exploration_summaries = (
         exp_services.get_exploration_summaries_matching_ids(exploration_ids))
+    displayable_exp_summaries = get_displayable_exp_summary_dicts(
+        exploration_summaries, exploration_ids)
+
+    return displayable_exp_summaries
+
+def get_displayable_exp_summary_dicts(exploration_summaries, exploration_ids):
+    """Given a list of exploration summary models and explorations
+    ids, returns a list of dicts of the corresponding exploration summaries
+    in displayable form"""
```

Comment:
Nit: add full stop at the end.

Also, modify the docstring:

    Given a list of exploration summary domain objects, return a list of the corresponding human-readable exploration summary dicts.